lib/ic/clock/ds3231.yaml

```diff
@@ -0,0 +1,17 @@
+name: DS3231MZ
+description: I2C integrated RTC/TCXO/Crystal
+datasheet: https://datasheets.maximintegrated.com/en/ds/DS3231M.pdf
+ordercodes: [["Farnell", "2515358"]]
+footprint: agg:SOIC-8
+pins:
+- - - [VBAT, 6, pwrin]
+    - [VCC, 2, pwrin]
+    - [GND, 5, pwrin]
+
+  - - [SDA, 7, od]
+    - [SCL, 8, in]
+
+- - - [32KHZ, 1, out]
+
+  - - [SQW/~INT, 3, od]
+    - [~RST, 4, od]
```

Comment:
Probably should be `bidi` as it's also usable as an input and also because it has an internal pullup so it's not really 'open drain'.